CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -14,65 +14,90 @@ Below are the rationales, notes, and caveats for this claim, organized by each r
 As an external package, `CycloneDDS` is not released on the same cadence as ROS.
 Changes to the branch of the `CycloneDDS` repository that is targeted by a given ROS release would be picked up by development builds of that ROS release.
-There is no published policy for what criteria are required for changes that are accepted in those branches.
 
-## Change Control Process [2]
+## Change Control [2]
+
+`CycloneDDS` follows the recommended guidelines of the [Eclipse Development Process](https://www.eclipse.org/projects/dev_process/).
+
+For the RMW layer, we follow the ROS core packages process. For Eclipse Cyclone DDS, we ensure the stability within the project through review, CI and tests and additionally run ROS CI for changes that are likely to affect ROS.
+
+All commits must be signed by the author and there must be an [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php) on file with the Eclipse Foundation.
+
+Pull requests are required to pass all tests in the CI system, unless Committers consider there is sufficient evidence that a failure is the result of a mishap unrelated to the change. Pull requests are only merged if the Committers deem it of acceptable quality and provide sufficient coverage of new functionality or proof of a bug fix via tests.
+
+Only the Committers have write access to the repository and they are voted for in elections monitored by the Eclipse Foundation staff.
 
 ### Change Requests [2.i]
 
-The source code repository for `CycloneDDS` is hosted in GitHub, and the change process follows a typical GitHub pull request model.
-There are several [team members](https://projects.eclipse.org/projects/iot.cyclonedds/who) with rights to merge the pull requests, and an election process exists for promoting new members of that team.
+All changes will occur through a pull request, check ROS 2 Developer Guide for additional information.
 
 ### Contributor Origin [2.ii]
 
-Perspective contributors to `CycloneDDS` must accept the terms of the [Eclipse Contributor Agreement](https://github.com/eclipse-cyclonedds/cyclonedds/blob/master/CONTRIBUTING.md#eclipse-contributor-agreement).
-This is confirmed by a signed-off-by line in the commits put forth for review, which is enforced by a GitHub plugin which blocks completion of the pull request until all contributions contain a sign-off.
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](https://www.eclipse.org/legal/DCO.php).
 
 ### Peer Review Policy [2.iii]
 
-There is no published policy for peer review of `CycloneDDS` contributions, but because all changes are made through a pull request, each requires an authorized committer to be merged.
+All pull requests will be peer-reviewed, check [Eclipse Developer Process](https://www.eclipse.org/projects/dev_process/) for additional information.
```

Comment:
Unfortunately this is self-evidently not the case — I wish there were meaningful reviews for all, but goal has proven elusive. Thus:

* Peer reviews are required for all pull requests from non-Committers and highly encouraged for all pull requests. The Committers are trusted to not merge any pull request without a review if there is a reasonably likelihood that such a review can be performed within a reasonable timeframe.